packages/backend-ai/src/types.ts

```diff
@@ -0,0 +1,197 @@
+export type JSONLike =
```

Comment:
Could we actually not roll our own types for building the Message content? - I.e. we just use the SDK's definitions for ContentBlock etc by using `import type { ContentBlock } from '@aws-sdk/client-bedrock-runtime`?

I think we need a source of truth between here and the data runtime client and since we're aiming to mimic their types anyway - maybe it makes sense to actually just use it